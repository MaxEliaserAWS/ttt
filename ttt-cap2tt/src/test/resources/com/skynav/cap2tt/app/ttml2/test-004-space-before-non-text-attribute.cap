Lambda字幕V4	DF0+1	SCENE"和文標準"

12	00002500/00002800	Align: ＠両端	＠縦右
				Justify
