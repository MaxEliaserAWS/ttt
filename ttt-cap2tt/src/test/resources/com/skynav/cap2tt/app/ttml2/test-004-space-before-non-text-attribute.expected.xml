<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttva:model="ttml2" xml:lang="ja" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" tts:textAlign="justify"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/><region xml:id="縦右" tts:extent="15rw 80rh" tts:position="right 10rw center" tts:writingMode="tbrl"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:00:25:00" end="00:00:28:00" region="縦右" style="s1">Align:<br/>Justify</p></div></body></tt>